src/modules/redesign/i18n/it_IT/verband/protokoll_types.ts

```diff
@@ -0,0 +1,20 @@
+export default {
+    mission: { regex: /Missione grande avviata/ },
+    event: { regex: /Evento alleanza avviato/ },
+    appl_accepted: { regex: /Aggiunto all'alleanza/ },
+    appl_declined: { regex: /Domanda respinta/ },
+    deny_appl: { regex: /Non autorizzato a presentare domanda per l'alleanza/ },
+    allow_appl: { regex: /Autorizzato a presentare domanda per l'alleanza/ },
+    left: { regex: /Lasciato l'alleanza/ },
+    kicked: { regex: /Cacciato dall'alleanza/ },
+    set_chatban: { regex: /Blocco chat impostato/ },
+    remove_chatban: { regex: /Blocco chat rimosso/ },
+    added_role: { regex: /Imposta come .*?/, title: 'Ruolo aggiunto' },
+    removed_role: { regex: /Rimuovi come .*?/, title: 'Ruolo rimosso' },
+    start_schooling: { regex: /Corso creato/ },
+    complete_schooling: { regex: /Corso completato/ },
+    start_extension: { regex: /Espansione avviata/ },
+    complete_extension: { regex: /Espansione finita/ },
+    build: { regex: /Edificio costruito/ },
+    demolish: { regex: /Edificio distrutto/ },
+};
```

Comment:
Please double check that the regexes match with the exact wordings used in alliance protocol!

Reply:
In this case I need to check the Alliance protocol wordings, where can I find it?

Reply:
https://www.operatore112.it/alliance_logfiles

It is only accessible by admins and the specific role.
If you don't got all values, don't worry. You can ask other players (maybe also we can look up some stuff). As redesign is not yet available on it_IT as far as I know, there's totally no problem to provide the correct values later.

Reply:
You mean admin of the alliance or of the game itself? Because if it's the admin of the alliance I can try to ask to my Alliance admin and check for that URL what is showing up...

Reply:
Alliance Admin is enough.

Reply:
I just asked my admins if is possible to have a screenshot of what's inside, now I have to wait...
Maybe to be more comfortable we can write to us on Discord privately if it's not a problem for you.